packages/core/src/utils/countable-set.ts

```diff
@@ -4,6 +4,8 @@ export class CountableSet<K> extends Set<K> {
   constructor(values?: Iterable<K>) {
     super(values)
     this._map ??= new Map()
+    for (const value of values ?? [])
```

Comment:
@antfu Why do we need this loop? `add()` method will be called twice for each value, because `super()` already does this under the hood.

Reply:
Because super does not add to `_map`

Reply:
But it calls our `add()` method which do this, isn't it?

Reply:
@antfu 
<img width="370" alt="Screenshot 2023-08-14 at 12 14 52" src="https://github.com/unocss/unocss/assets/10506522/f2615528-9080-4e2a-8ba5-5ede6abc5f86">

Reply:
We could add some test and remove it if not necessary